devices/generic/subdevices/color_temperature_light.json

```diff
@@ -6,22 +6,26 @@
     "order": 11,
     "uuid": ["$address.ext", "0x01"],
     "items": [
+        "cap/color/ct/max",
+        "cap/color/ct/min",
+        "state/alert",
+        "state/bri",
+        "state/colormode",
+        "state/ct",
+        "state/on",
+        "state/reachable"
+    ],
+    "items_optional":[
         "cap/alert/trigger_effect",
         "cap/bri/move_with_onoff",
         "cap/color/capabilities",
```

Comment:
`cap/color/capabilities` should be mandatory.

Reply:
Done